docs/source/running_mypy.rst

```diff
@@ -182,7 +182,7 @@ corresponding type hints.
 
 Mypy will not try inferring the types of any 3rd party libraries you have installed
 unless they either have declared themselves to be
-:ref:`PEP 561 compliant stub package <installed-packages>` or have registered
+:ref:`PEP 561 compliant stub package <installed-packages>` with a ``py.typed`` file or have registered
```

Comment:
Actually not all installed packages need a `py.typed` file -- this marker is only needed when the `.pyi` files are in the same directory as the `.py` files. For packages containing only stubs, to be installed separately, no `py.typed` file is needed; instead such packages are recognized by the name `...-stubs`.

Read PEP 561 about it: https://www.python.org/dev/peps/pep-0561/#stub-only-packages